actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -179,6 +179,14 @@ def method_missing(called, *args, **options, &block)
       #   tag.div data: { city_state: %w( Chicago IL ) }
       #   # => <div data-city-state="[&quot;Chicago&quot;,&quot;IL&quot;]"></div>
       #
+      # ARIA Array attribute values are space separated to be treated as <tt>DOMTokenList</tt> values.
```

Comment:
I am a bit concerned about the change in behavior here not being quite backward compatible.

However, I don't see any mention of treating `aria-` the way we treat `data-` in the documentation, so maybe it was a private API before (🤷  😉 ).